Makefile

```diff
@@ -1,7 +1,9 @@
 all: install
 install: clean
+	#on python 2.6, pre-install last supported version of numpy
+	python --version 2>&1 >/dev/null | grep '2.6' && pip install 'numpy==1.10.0' || true
 	python setup.py sdist
-	pip install dist/*.tar.gz --upgrade
+	pip install dist/*.tar.gz --upgrade --upgrade-strategy=only-if-needed
```

Comment:
Could we make this 'only-if-needed' strategy also python2.6 depend ? I don't really like the fact that python2.6 would be blocking us from using the latest versions of everything in the other python versions